int64 a
---
int64 result
